PhilipFunkos/src/App.jsx

```diff
@@ -0,0 +1,53 @@
+import { useState } from "react";
+import reactLogo from "./assets/react.svg";
+import "./App.css";
+import Home from "./home";
+import Product from "./products";
+import Contact from "./contact";
+
+function App() {
+  function home() {
+    let page1 = document.querySelector(".home-page");
```

Comment:
You dont need to create a variable if its only used once. Also, if it does not change it should be a `const`.